CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+Dont think this is relevant to us because we dont run tests and i have talked about instalation/prerequisites in readme 
+???????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????
+
+## How contributors can get started if they are newcomers
+
+??????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????? 
+Dont think this is relevant to us because we have pre defined tasks and I am not sure which ones would be best for newcomers
+???????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????
+
+## Technical Requirements for Contributions
+
+Our backend logic is all in java so please use java and not kotlin!
+
+We also value having a consistent style in our program, this is to minimise confusion and keep everything looking neat and consistent! As such we will be following
+googles style guide for java programs shown at this link: https://google.github.io/styleguide/javaguide.html
+
+## Our Vision
+
+We have put a lot of thought into how we want RondayView to move forward with features, improvements and awesomeness from start to finish with some checkpoints 
+in between. We have split the app up into 3 stages and called them A1, A2 and A3, with each stage building upon the former. As such, we would like all features
+of A1 to be completed before starting features of A2, and the same for A2 going to A3. I have listed these milestone features below but they are also shown in 
+the Issues tab. 
+
+For A1, we would like to cover basic functions of the app. These include the following:
+* Create Events – anyone can create events initially and this is stored in database
+* Browse Events – anyone can browse events
+* Swipe events to show interest or disinterest
+* Basic Filter - event dates, event size, virtual or in person
+* Basic initial user interface
+* Profiles - Local storage / Favourites
+
+Moving to A2, we would like to build mostly upon the social aspects of the program. The features that we would like to implement are as follows:
+* User profiles - Sign in, accounts stored in databases
+* Able to add other user profiles as friends
+* View events that friends are going to
+* View popular events among all users
+* Leaderboard aspect (who has visited the most events)
+
+Finally, at A3, we would like to add features that integrate the project into other aspects of the phone, as well as further cater the app towards individual users with the following features:
+* In-app and push notifications for upcoming events
+* Link to the phones calendar
+* Advanced Filters - Tags / event type, event host name
+* Each event gets a photo album that attendees can add to
+* Event rating and comments – previous events rating too
+* Suggested similar events in event description
+* Private Groups and private events
+
```

Comment:
It could be good to add it here to add that features are detailed more in GitHub issues and to contact us if they are keen on helping on a particular feature